WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduleFormatter.swift

```diff
@@ -13,44 +13,68 @@ struct BloggingRemindersScheduleFormatter {
 
     /// Attributed description string of the current schedule for the specified blog.
     ///
-    func shortIntervalDescription(for schedule: BloggingRemindersScheduler.Schedule) -> NSAttributedString {
+    func shortScheduleDescription(for schedule: BloggingRemindersScheduler.Schedule, time: String? = nil) -> NSAttributedString {
         switch schedule {
         case .none:
             return Self.stringToAttributedString(TextContent.shortNoRemindersDescription)
         case .weekdays(let days):
             guard days.count > 0 else {
-                return shortIntervalDescription(for: .none)
+                return shortScheduleDescription(for: .none, time: time)
             }
 
-            return Self.shortIntervalDescription(for: days.count)
+            return Self.shortScheduleDescription(for: days.count, time: time)
         }
     }
 
-    static func shortIntervalDescription(for days: Int) -> NSAttributedString {
+    static func shortScheduleDescription(for days: Int, time: String?) -> NSAttributedString {
+        guard let time = time else {
+            return shortScheduleDescription(for: days)
+        }
+        return shortScheduleDescriptionWithTime(for: days, time: time)
+    }
+
+    static func shortScheduleDescriptionWithTime(for days: Int, time: String) -> NSAttributedString {
+        let text: String = {
+            switch days {
+            case 1:
+                return String(format: TextContent.oneReminderShortDescriptionWithTime, time)
+            case 2:
+                return String(format: TextContent.twoRemindersShortDescriptionWithTime, time)
+            case 7:
+                return "<strong>" + String(format: TextContent.everydayRemindersShortDescriptionWithTime, time) + "</strong>"
+            default:
+                return String(format: TextContent.manyRemindersShortDescriptionWithTime, days, time)
+            }
+        }()
+
+        return Self.stringToAttributedString(text)
+    }
+
+    static func shortScheduleDescription(for days: Int) -> NSAttributedString {
+
```

Comment:
What do you think about moving these into an extension?

Reply:
Totally, will do now!